### PR TITLE
0.7.2 release

### LINKED ISSUE #568: Parse VERSION to determine if a github release is a pre-release
Currently, all github releases are set to pre-releases. We should move this parse the VERSION string to see if there's a suffix regex of `-(alpha|beta).[0-9]`, this is a pre-release, if this doesn't end with that regular expression, we should publish it as a full release.

### PR DESCRIPTION
**Pull request by simlay**
Releases 0.7.2 and closes #568.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.7.2-alpha.0
\ No newline at end of file
+0.7.2-alpha.1
```
Comment (sehz) [inline VERSION:1]: I assume this is for testing release workflow?

**Review by sehz — CHANGES_REQUESTED** [review]

File: VERSION (hunk 1 above)
Comment (simlay) [reply VERSION:1]: Yup. I'm doing a pre-release test now.

**Review by sehz — APPROVED** [review]

File: makefiles/S3.toml [hunk 2]
```diff
@@ -63,3 +63,7 @@ script = """${HOME}/.fluvio/bin/fluvio package --test bump dynamic ${FLUVIO_VERS
 [tasks.bump-fluvio-latest]
 dependencies = ["install-fluvio-package"]
 script = """${HOME}/.fluvio/bin/fluvio package --test bump latest "$(cat VERSION)+$(git rev-parse HEAD)" """
+
+[tasks.bump-fluvio-stable]
+dependencies = ["install-fluvio-package"]
+script = """${HOME}/.fluvio/bin/fluvio package --bump stable $(cat VERSION)"""
```
Comment (nicholastmosher) [inline makefiles/S3.toml:69]: This should be `fluvio package bump stable $(cat VERSION)`, not `--bump`

Comment (simlay) [reply makefiles/S3.toml:69]: Ah yes, good catch.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -49,13 +49,13 @@ jobs:
...
         run: |
-          echo "::set-env name=AWS_ACCESS_KEY_ID::${{ secrets.PACKAGE_TEST_AWS_ACCESS_KEY_ID }}"
-          echo "::set-env name=AWS_SECRET_ACCESS_KEY::${{ secrets.PACKAGE_TEST_AWS_SECRET_ACCESS_KEY }}"
+          echo "AWS_ACCESS_KEY_ID=${{ secrets.PACKAGE_TEST_AWS_ACCESS_KEY_ID }}" >> $GITHUB_ENV
+          echo "AWS_SECRET_ACCESS_KEY=${{ secrets.PACKAGE_TEST_AWS_SECRET_ACCESS_KEY }}" >> $GITHUB_ENV
       - name: Set prod env
         if: ${{ github.event.inputs.test == '' }}
         run: |
-          echo "::set-env name=AWS_ACCESS_KEY_ID::${{ secrets.AWS_ACCESS_KEY_ID }}"
-          echo "::set-env name=AWS_SECRET_ACCESS_KEY::${{ secrets.AWS_SECRET_ACCESS_KEY }}"
```
Comment (nicholastmosher) [inline .github/workflows/release.yml:58]: Dang, sorry that was my bad. I thought the ::set-env stuff was something I'd only experimented with in my test-release branch, I didn't know that had found its way into master :(

Comment (simlay) [reply .github/workflows/release.yml]: No worries.

**Pull request merged**